modules/nextflow/src/main/resources/nextflow/trace/assets/ReportTemplate.js

```diff
@@ -166,6 +166,7 @@ $(function() {
     if (bytes == '-' || bytes == 0){
       return bytes;
     }
+    bytes = norm_mem([bytes]);
     // https://stackoverflow.com/a/14919494
     var thresh = 1000;
```

Comment:
Also, using `thresh = 1024` should not do the same instead of using `norm_mem` ?